### PR TITLE
AF-1799: Upgrade JDK 11 + GWT 2.9

### PR DESCRIPTION
**Pull request by caponetto**
**Upgrade JDK 11 + GWT 2.9**

Feel free to comment and/or raise any concerns.

This PR continues the work done in 
- https://github.com/kiegroup/appformer/pull/1030
- https://github.com/kiegroup/appformer/pull/1039

Thanks guys for that!

**JIRA**:

[AF-1799](https://issues.redhat.com/browse/AF-1799)

**Parent Pull Request**:
- https://github.com/kiegroup/lienzo-core/pull/134

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by caponetto** [comment]
Jenkins run fdb
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run cdb

**Comment by caponetto** [comment]
Jenkins run fdb

**Comment by caponetto** [comment]
Jenkins run fdb

**Comment by caponetto** [comment]
Jenkins run fdb

**Comment by caponetto** [comment]
Jenkins run fdb

**Comment by caponetto** [comment]
Jenkins retest this

**Comment by caponetto** [comment]
Jenkins retest this

**Comment by caponetto** [comment]
Jenkins retest this

**Comment by caponetto** [comment]
Jenkins retest this

**Comment by caponetto** [comment]
Jenkins retest this

File: jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/migration/MigrationManager.java [hunk 1]
```diff
@@ -389,6 +389,9 @@ private void upgradeProcessInstance(KieRuntime oldkruntime,
         if (process == null) {
             report.addEntry(Type.ERROR, "Could not find process " + processId);
         }
+        if (processInstance == null) {
+            return;
+        }
```
Comment (MarianMacik) [inline jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/migration/MigrationManager.java:394]: Is this change needed for the upgrade? In case we want to avoid NPE, I would suggest to check if there is any error in the report and if so, just return. This is just a partial solution because if `process` or `processId` is null, then we will continue. So we should stop in case any of the three is null.

In case it is not needed for the upgrade, I would solve it in a separate PR.

Comment (caponetto) [reply jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/migration/MigrationManager.java:394]: Hi @MarianMacik, I remember that I've got a complaint from the building process due to the lack of a null check there. If `processInstance` is null, then the next line will break (`processInstance.getProcessId().equals(processId)`).
Anyway, I removed locally this check and the build passed so I'll revert this change. Let's see if we get a green build from Jenkins.

**Review by MarianMacik — APPROVED** [review]
Thanks. This check is indeed an improvement, though I wouldn't mix it here.

**Comment by caponetto** [comment]
Jenkins retest this

**Review by elguardian — APPROVED** [review]

**Comment by elguardian** [comment]
@mareknovotny should I merge this or you take the coordination of every PR associated to this one ?

**Comment by mareknovotny** [comment]
@elguardian approval is enough from you. We need to coordinate all PRs merging related to this jdk11/gwt upgrade

**Review by Srihari1192 — APPROVED** [review]
